paper/paper.md

```diff
@@ -0,0 +1,150 @@
+---
+title: 'Pyglmnet: Python implementation of elastic-net regularized generalized linear models'
+tags:
+  - Python
+  - glm
+  - machine-learning
+  - lasso
+  - elastic-net
+authors:
+  - name: Mainak Jas
+    orcid: 0000-0003-0872-7098
+    affiliation: "1, 2"
+  - name: Pavan Ramkumar
+    affiliation: 3
+  - name: Hugo Fernandes
+    affiliation: 10
+  - name: Daniel Acuna
+    affiliation: 6
+  - name: Ravi Garg
+    affiliation: 5
+  - name: Aid Idrizović
+    affiliation: 4
+  - name: Vinicius Marques
+    affiliation: 4
+  - name: Tommy Odland
+    affiliation: 13
+  - name: Giovanni De Toni
+    affiliation: 14
+  - name: Peter Foley
+    affiliation: 11
+  - name: Chris Rodgers
+    affiliation: 8
+  - name: Beibin Li
+    affiliation: 12
+  - name: Eva Dyer
+    affiliation: 9
+  - name: Matti Hamalainen
+    affiliation: "1, 2"
+  - name: Konrad Kording
+    affiliation: 5
+affiliations:
+ - name: Massachusetts General Hospital
+   index: 1
+ - name: Harvard Medical School
+   index: 2
+ - name: Balbix
+   index: 3
+ - name: Unknown
+   index: 4
+ - name: University of Pennsylvania
+   index: 5
+ - name: University of Syracuse
+   index: 6
+ - name: Columbia University
+   index: 8
+ - name: Georgia Tech
+   index: 9
+ - name: Rockets of Awesome
+   index: 10
+ - name: 605
```

Comment:
can we find the exact company name?

Reply:
that is the company name :) But I believe @peterfoley will have a new affiliation soon

Reply:
I think my name and affiliation looks good! Should I just check the box in the PR?

Reply:
yes you can!